src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/ConversionKind.cs

```diff
@@ -12,7 +12,12 @@ internal enum ConversionKind : byte
         Identity,
         ImplicitNumeric,
         ImplicitEnumeration,
-        ImplicitTuple,
+
+        /// <summary>
+        /// An implicit conversion from a tuple literal to a tuple or a tuple compatible type of matching cardinality.
```

Comment:
Can you clarify what hte second case means? i.e. hte part about "a tuple compatible type of matching cardinality".  What's a code example where you get this?  Thanks!